plugins/index.json

```diff
@@ -14,7 +14,7 @@
   {
     "name": "kubernetes",
     "author": "Porter Authors",
-    "description": "Integrates Porter with Kubernetes. Store Porter's data and your bundle's secrets in Kubernetes Secret Resources.",
+    "description": "Integrates Porter with Kubernetes. Let you secure your bundle's secrets in Kubernetes Secret Resources.",
```

Comment:
nit: "Secure your bundle's secrets in Kubernetes Secret Resources" reads a little better IMO

Reply:
Agree, changed